Dockerfile

```diff
@@ -1,29 +1,20 @@
 #################################
-# Botiana Dockerfile for Debian #
+# Botiana Dockerfile for Alpine #
 #################################
-
-FROM debian
+FROM python:3.7.4-alpine3.10
 MAINTAINER rpkish
 
-# Configure the basic debian image
-RUN apt-get update && apt-get install -y \
-    aufs-tools \
-    python \
-    python-pip \
-    python-lxml \
-    filters \
-    git \
-    python-yaml \
- && rm -rf /var/lib/apt/lists/* \
- && pip install BeautifulSoup \
- && pip install beautifulsoup4 
+# Build deps
+RUN apk update \
+    && apk add --virtual build-dependencies build-base py3-lxml libxml2-dev libxslt-dev
 
 # Install botiana and copy modified settings and dictionaries 
 RUN mkdir /usr/local/botiana/
```

Comment:
iirc `COPY` will make one for you implicitly

Reply:
Dockerfile updated. Thanks!